drydock_backups/plugin.py

```diff
@@ -8,12 +8,13 @@
 import pkg_resources
 from tutor import hooks
 
-from .__about__ import __version__
+from drydock.__about__ import __version__
 
+DRYDOCK_IMAGE = f"ednxops/shipyard-utils:v{__version__}"
 config = {
     "defaults": {
         "VERSION": __version__,
-        "IMAGE": "ednxops/shipyard-utils:v15.4.0",
+        "IMAGE": DRYDOCK_IMAGE,
```

Comment:
```suggestion
        "DOCKER_IMAGE": "ednxops/shipyard-utils:v{{ BACKUP_VERSION }}",
```
Better to interpolate as done [here](https://github.com/hastexo/tutor-contrib-backup/blob/main/tutorbackup/plugin.py#L16). Do you consider changing BACKUP_IMAGE to BACKUP_DOCKER_IMAGE would have a big impact. What manifests would we need to modify?

Reply:
It is not possible to use `{{BACKUP_VERSION}}` since the variable is created from `VERSION` after defining the default variables, unless it is defined in `config.yml`, which seems redundant if `DOCKER_IMAGE` is defined.

List of manifests using `drydock-backups`
- [manifests-bluenose monbasa](https://github.com/eduNEXT/manifests-bluenose/blob/main/environments/mombasa/config.yml#L9-L19)
- [manifests-aeolus cagliari](https://github.com/eduNEXT/manifests-aeolus/blob/main/environments/cagliari/config.yml#L1-L11)
- [manifests-achille-lauro accra](https://github.com/eduNEXT/manifests-achille-lauro/blob/main/environments/accra/config.yml#L1-L6)
- [manifests-admiral alicante](https://github.com/eduNEXT/manifests-admiral/blob/main/environments/alicante/config.yml#L1-L10)
- [manifests-la-belle pusan](https://github.com/eduNEXT/manifests-la-belle/blob/main/environments/pusan/config.yml#L1-L10)
- [manifests-pilar paita](https://github.com/eduNEXT/manifests-pilar/blob/main/environments/paita/config.yml#L1-L10)
- [manifests-oneida montevideo](https://github.com/eduNEXT/manifests-oneida/blob/main/environments/montevideo/config.yml#L1-L10)
- [manifests-griswold grimsby](https://github.com/eduNEXT/manifests-griswold/blob/main/environments/grimsby/config.yml#L1-L10)

no manifest defines the variable `IMAGE`